logs/apiDebug.log

```diff
@@ -0,0 +1,4 @@
+{"level":"info","message":"request recieved","timestamp":"2017-11-14T10:40:41.042Z"}
```

Comment:
Please add this file (logs/apiDebug.log) to .gitignore and remove from repository